### PR TITLE
web-ui: Add supported rules page with filtering

### LINKED ISSUE #300: Show supported rules in web-ui
After merging #275 we can create a page where all enabled rules are listed.

### PR DESCRIPTION
**Pull request by Nagogus**
web-ui: Add supported rules page with filtering

Closes #300

### CODE REVIEW

**Comment by Nagogus** [comment]
@rbarilani

File: web-ui/src/client/app/services/rest.js [hunk 1]
```diff
@@ -30,6 +30,29 @@ export const RestService = {
...
+  },
+  getApiSupportedRules (params) {
+    const options = {
+      method: 'GET',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      }
+    };
+    const query = params ? '?' + Object.keys(params)
```
Comment (rbarilani) [inline web-ui/src/client/app/services/rest.js]: @Nagogus can we extract this "object to params" functionality to a function...

File: web-ui/src/client/app/services/__tests__/rest.test.js [hunk 2]
```diff
@@ -81,5 +81,43 @@ describe('RestService', () => {
...
+  });
+
+  test('getApiSupportedRules call api with filter query and resolve with response body representing rules', () => {
+    const mockRules = [];
+    const rulesResponse = {
+      json:() => mockRules
+    };
+    client.fetch.mockReturnValueOnce(Promise.resolve(rulesResponse));
+
+    return RestService.getApiSupportedRules({'is_active': true}).then((rules) => {
```
Comment (rbarilani) [inline web-ui/src/client/app/services/__tests__/rest.test.js]: @Nagogus can we call this method just ```getSupportedRules```

File: web-ui/src/client/app/containers/rules.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+export class Rules extends Component {
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      error: null,
+      pending: false,
+      ajaxComplete: false,
+      fetchActiveRules: true,
+      filter: this.parseFilterValue(this.props.location.query),
```
Comment (rbarilani) [inline web-ui/src/client/app/containers/rules.jsx]: @Nagogus Maybe better to move this in the ```componentDidMount```

File: web-ui/src/client/app/components/rules.jsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+export function Rule (props) {
+  const rule = props.rule;
+  return (
+    <li style={{marginBottom: '32px', paddingBottom: '32px', borderBottom: '1px solid #ccc'}}>
+      <h4 className="dc-h4">
+        <RuleType type={rule.type} />
+        {rule.type} {'\u2013'} {rule.title}
+      </h4>
+      <If test={() => !!rule.url} dataTestId="if-rule-url" >
```
Comment (rbarilani) [inline web-ui/src/client/app/components/rules.jsx]: @Nagogus Use the ternary operator here.... it's simplier to test...

**Comment by Nagogus** [comment]
@rbarilani changes applied

**Comment by rbarilani** [comment]
👍

**Comment by netme** [comment]
👍

**Pull request merged**